inithome/requirements/dev.in

```diff
@@ -0,0 +1,37 @@
+# -*- conf -*-
+#
+# Editable development dependencies for the Nublado controller. Add direct
+# development, test, and documentation dependencies here, as well as
+# implicit dev dependencies with constrained versions.
+#
+# After editing, update requirements/dev.txt by running:
+#     nox -s update-deps
+
+-c main.txt
+
+# Testing
+pytest
+pytest-cov
+pytest-asyncio
+pyfakefs
+# The top-level pyproject.toml configures the Pydantic mypy plugin, so
+# pydantic has to be installed for mypy to run.  We don't use pydantic
+# ourselves in inithome.
+pydantic
+
+
+
+asgi-lifespan
+coverage[toml]
+httpx-sse
+pytest
+pytest-asyncio
+pytest-cov
+pytest-sugar
+respx
+types-pyyaml
+
+# Documentation
+autodoc_pydantic
+documenteer[guide]>=1.0.0a7
+sphinx-diagrams
```

Comment:
Looks like some left-over cut and paste that can be removed.

Reply:
Removed.